src/plugins/intel_cpu/src/nodes/mvn.cpp

```diff
@@ -310,103 +296,65 @@ void MVN::prepareParams() {
-        if (mvnAttrs.layout == MVNLayoutType::mvn_by_channel) {
-            mvnAttrs.actualChannelSize = shape5D[shape5D.size() - 1];
-        } else {
-            mvnAttrs.actualChannelSize = shape5D[1];
-        }
+        mvnAttrs.actualChannelSize = shape5D[1];
     } else if (shape5D.size() == 1) {
         // For 1D case, the entire dimension might be channels
         mvnAttrs.actualChannelSize = mvnAttrs.initAcrossChannels_ ? 1 : shape5D[0];
     } else {
         mvnAttrs.actualChannelSize = 1;
     }
 
-    // Populate post-ops from fused nodes
+    // Populate post-ops from fused nodes (no adjustments)
     mvnAttrs.postOps = getPostOps(fusedWith);
 
-    // Special handling for Instance Normalization pattern
-    // When MVN is followed by Multiply/Add with [1,C,1,1] shape tensors,
-    // we need to handle them specially to avoid dimension mismatch
-    PostOps adjustedPostOps;
-    for (const auto& postOp : mvnAttrs.postOps) {
-        try {
-            if (postOp.type() == typeid(ScaleShiftPostOp)) {
-                const auto& scaleShiftOp = std::any_cast<const ScaleShiftPostOp&>(postOp);
-
-                std::vector<float> adjustedScales = scaleShiftOp.scales();
-                std::vector<float> adjustedShifts = scaleShiftOp.shifts();
-
-                // Check if scales have broadcasting pattern [1,C,1,1]
-                if (adjustedScales.size() > 1 && adjustedScales.size() != mvnAttrs.actualChannelSize) {
-                    // This is likely a [1,C,1,1] pattern - use scalar broadcast
-                    float firstScale = adjustedScales[0];
-                    adjustedScales.clear();
-                    adjustedScales.push_back(firstScale);
-                }
-
-                // Check if shifts have broadcasting pattern [1,C,1,1]
-                if (adjustedShifts.size() > 1 && adjustedShifts.size() != mvnAttrs.actualChannelSize) {
-                    // This is likely a [1,C,1,1] pattern - use scalar broadcast
-                    float firstShift = adjustedShifts[0];
-                    adjustedShifts.clear();
-                    adjustedShifts.push_back(firstShift);
-                }
+    // Update executor with memory arguments
+    MemoryArgs memoryArgs;
+    memoryArgs[ARG_SRC_0] = getSrcMemoryAtPort(0);
+    memoryArgs[ARG_DST] = getDstMemoryAtPort(0);
 
-                // Create new ScaleShiftPostOp with adjusted values
-                adjustedPostOps.push_back(
-                    std::make_any<ScaleShiftPostOp>(scaleShiftOp.type(), adjustedScales, adjustedShifts));
-            } else {
-                // Keep other post-ops as is
-                adjustedPostOps.push_back(postOp);
-            }
-        } catch (const std::bad_any_cast&) {
-            // Not a ScaleShiftPostOp, keep as is
-            adjustedPostOps.push_back(postOp);
-        }
+    {
+        auto execCtx = std::make_shared<ExecutorContext>(context, getImplPriority());
+        auto makeFactory =
```

Comment:
In scope of prepare params method we are only supposed to 'update' an already created executor.
You could look at other nodes as an example.
Why do we have to recreate everything here?